### PR TITLE
Added launchSettings.json for Blazor wasm templates

### PR DESCRIPTION
**Pull request by ajaybhargavb**
Fixes https://github.com/dotnet/aspnetcore/issues/17443

### CODE REVIEW

**Review by ryanbrandenburg — APPROVED** [review]

**Review by mkArtakMSFT — APPROVED** [review]

File: src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Client/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "useWebAssemblyDebugging": true
+}
```
Comment (javiercn) [inline src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Client/Properties/launchSettings.json]: @ajaybhargavb I don't think we need it on the client template.

File: src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Server/Properties/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "useWebAssemblyDebugging": true
+}
```
Comment (javiercn) [inline src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Server/Properties/launchSettings.json]: Can you start from an existing launchSettings template? This one is missing all the config, if you are adding it, keep it in sync with the rest :)

Comment (ajaybhargavb) [reply src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Server/Properties/launchSettings.json]: Sure

File: src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Client/Properties/launchSettings.json (hunk 1 above)
Comment (ajaybhargavb) [reply src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Client/Properties/launchSettings.json]: We need this config for Blazor standalone apps as well.

**Comment by ajaybhargavb** [comment]
🆙 📅

File: src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Server/Properties/launchSettings.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+    "useWebAssemblyDebugging": true,
+    "iisSettings": {
+      //#if (WindowsAuth)
```
Comment (javiercn) [inline src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Server/Properties/launchSettings.json]: This option is not supported on the wasm template yet.

**Review by javiercn — APPROVED** [review]
Looks great!

Merge once you make the suggested change.

**Pull request merged**